Jellyfin.Server.Implementations/FullSystemBackup/BackupService.cs

```diff
@@ -193,7 +193,7 @@ void CopyDirectory(string source, string target)
                     {
                         _logger.LogInformation("Read backup of {Table}", entityType.Type.Name);
 
-                        var zipEntry = zipArchive.GetEntry($"Database\\{entityType.Type.Name}.json");
+                        var zipEntry = zipArchive.GetEntry(Path.Join("Database", $"{entityType.Type.Name}.json"));
```

Comment:
There is a mix of Path.Join and Path.Combine and It's not clear why sometimes one is used over the other.